### PR TITLE
ENH: Add instructions for setting up ipython profiles

### PR DESCRIPTION
**Pull request by ericdill**
@danielballan not sure if this is the best spot for these docs. Let me know if you want them somewhere else.

### CODE REVIEW

**Comment by tacaswell** [comment]
This is going to get overhauled very soon to install everything via conda packages.

File: source/install.rst [hunk 1]
```diff
@@ -124,3 +124,41 @@ Upgrade
...
+      c.TerminalIPythonApp.pylab = 'auto'
+
+#. Copy over the README.md file from `here <https://raw.githubusercontent.com/NSLS-II-CSX/ipython_ophyd/master/profile_xf23id1/startup/README>`_
+   to ``$IPYTHONDIR/profile_ophyd/startup/README.md``
+
+#. Running ``ipython --profile=ophyd`` should now successfully start up ophyd.
+   To check that it is working, run `wh_pos` at the ipython prompt and make
+   sure that an error is not thrown.
+
+#. To start adding positioners and detectors, see the currently activate
```
Comment (danielballan) [inline source/install.rst]: activate -> active

**Comment by danielballan** [comment]
@tacaswell True, but the last bit (examples) is going to give us somewhere to point people.

@ericdill This seems like a good place to me.

**Comment by ericdill** [comment]
@tacaswell Great!  The primary reason that I wrote these docs was because @cmazzoli came to me and asked how to set up an ophyd ipython profile for CSX-2 so that they could start commissioning the other branch and I had no idea how to do that for him.

**Comment by ericdill** [comment]
If there are no objections, could this get merged?

**Comment by ericdill** [comment]
Woo thanks @danielballan

**Pull request merged**